pom.xml

```diff
@@ -66,10 +66,10 @@
         <changelist>-SNAPSHOT</changelist>
         <jenkins.version>2.150.1</jenkins.version>
         <java.level>8</java.level>
-        <workflow-step-api-plugin.version>2.22</workflow-step-api-plugin.version>
-        <workflow-cps-plugin.version>2.70</workflow-cps-plugin.version>
+        <workflow-step-api-plugin.version>2.23-SNAPSHOT</workflow-step-api-plugin.version>
+        <workflow-cps-plugin.version>2.81-SNAPSHOT</workflow-cps-plugin.version>
```

Comment:
Just for future reference, if you have other PRs that are linked to a PR, please include a link to them in the PR description. Instead of using SNAPSHOT dependencies, you can use [incremental dependencies](https://jenkins.io/blog/2018/05/15/incremental-deployment/), for example [here](https://repo.jenkins-ci.org/incrementals/org/jenkins-ci/plugins/workflow/workflow-step-api/2.23-rc561.8c6752584d39/) is the incremental build for https://github.com/jenkinsci/workflow-step-api-plugin/pull/55.

Reply:
Hello @dwnusbaum , thanks for reviewing this and for your suggestion. I closed the issue because setting catchInterruptions to false works.
Thanks,
Best regards,